geometry_msgs/Point p1
geometry_msgs/Point p2
geometry_msgs/Point p3
geometry_msgs/Point p4
float64 yaw
uint32 parking_space_type
float64 length
float64 width
